rtshell/rtcon.py

```diff
@@ -134,6 +134,12 @@ def property_callback(option, opt, option_value, parser):
         print('{0}: No ports specified.'.format(os.path.basename(sys.argv[0])),
                 file=sys.stderr)
         return 1
+
+    if len(args) < 2:
```

Comment:
Fixed(reverted) in this commit.

https://github.com/yosuke/rtshell/commit/adee1aea216e3772db87351d03380bea77147060